storm-server/src/main/java/org/apache/storm/container/cgroup/CgroupManager.java

```diff
@@ -203,6 +209,30 @@ public void reserveResourcesForWorker(String workerId, Integer totalMem, Integer
             }
         }
 
+        if (numaId != null) {
```

Comment:
Is this a different implementation than what we have internally? 
Internally you used `numactl`. Here seems to be using cgroup directly. Is this well tested?